From 08b10ca2e2de965fcd3ba90c3229f4ff3ff2d531 Mon Sep 17 00:00:00 2001
From: Philippe Gerum <rpm@xenomai.org>
Date: Sat, 20 Jun 2020 15:39:57 +0200
Subject: [PATCH 0642/1013] include/evl: wait: do not drag tracepoints in

Core EVL headers should be readable from the asm-generic/ section
(e.g. evl/wait.h), which among other things requires the tracepoint
definitions not to be pulled in. The latter should be read from the
files implementating them instead.

Signed-off-by: Philippe Gerum <rpm@xenomai.org>
---
 drivers/evl/hectic.c | 1 +
 include/evl/wait.h   | 1 -
 2 files changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/evl/hectic.c b/drivers/evl/hectic.c
index b20c483d38ed..fd384e0672b8 100644
--- a/drivers/evl/hectic.c
+++ b/drivers/evl/hectic.c
@@ -19,6 +19,7 @@
 #include <evl/stax.h>
 #include <asm/evl/fptest.h>
 #include <uapi/evl/devices/hectic.h>
+#include <trace/events/evl.h>
 
 #define HECTIC_KTHREAD      0x20000
 
diff --git a/include/evl/wait.h b/include/evl/wait.h
index d8f27f7ed166..813c8d5c4026 100644
--- a/include/evl/wait.h
+++ b/include/evl/wait.h
@@ -15,7 +15,6 @@
 #include <evl/list.h>
 #include <evl/timer.h>
 #include <evl/clock.h>
-#include <trace/events/evl.h>
 #include <uapi/evl/thread.h>
 
 #define EVL_WAIT_FIFO    0
-- 
2.30.2

